test/Prompt/DontWrap.C

```diff
@@ -172,7 +172,7 @@ const Test2::Nested::Value::type & Test2::Nested::reference() const {
 Test2::Nested Nest2(80);
 // CHECK: Nested::Nested(160)
 Nest2.simpleAdd(3)
-// CHECK: (Test2::Nested::Value::type) 83
+// CHECK: (int) 83
```

Comment:
In the ROOT code we have a function ROOT::TMetaUtils::GetUnderlyingType which giving a type returns the end node (aka the 'raw' type, aka the underlying type).   For example for "size_t**" it return "long" (on platform where size_t is a long).

I wonder if we can import this routine and then test whether the underlying type is a clang::BuiltinType.